### ISSUE TITLE
How can I get the train.txt and val.txt for COESOT?

### ISSUE THREAD

**Issue by ruihuixu** [Event-AHU/COESOT#18]
When I run coesot.py in lib/train/dataset, I encountered an issue with the missing list.txt. Looking at previous issues, I understand it's related to train.txt and val.txt, but I couldn't find these two files. Could you give me some hints about getting train.txt and val.txt? Thank you very much

**Comment by Tchuanm** [comment]
Hi, For this issue, you can check https://github.com/Event-AHU/COESOT/issues/2 which can help you to find the difference of train, val and test.txt.

**Comment by ruihuixu** [comment]
> Hi, For this issue, you can check [#2](https://github.com/Event-AHU/COESOT/issues/2) which can help you to find the difference of train, val and test.txt.嗨，对于这个问题，您可以查看 [#2](https://github.com/Event-AHU/COESOT/issues/2) 哪些可以帮助您找到 train、val 和 test.txt 的差异。

I have already checked that, but the problem is that I can't find the train.txt, val.txt, and test.txt files. Currently, I have randomly selected 40 sequences from the training_subset as the validation set (val.txt), and the rest are used as the training set (train.txt). Is this approach correct?Thank you for your answer.

**Comment by wangxiao5791509** [comment]
The training and testing subset can be obtained via: 

https://github.com/Event-AHU/COESOT/tree/main/COESOT_eval_toolkit/annos/coesot_anno

https://github.com/Event-AHU/COESOT/blob/main/COESOT_eval_toolkit/sequence_evaluation_config/COESOT_testing_subset.txt 

@Tchuanm @ruihuixu

**Comment by wangxiao5791509** [comment]
We split sequences into a training and testing subset, which contains 827 and 527 videos, respectively. @Tchuanm @ruihuixu

**Comment by ruihuixu** [comment]
Thanks！

**Comment by wangxiao5791509** [comment]
The txt file for these subset can be found below (Newly updated): 

https://github.com/Event-AHU/COESOT/blob/main/COESOT_train.txt
https://github.com/Event-AHU/COESOT/blob/main/COESOT_val.txt
https://github.com/Event-AHU/COESOT/blob/main/COESOT_test.txt
